plugin/src/main/cpp/extensions/openxr_fb_composition_layer_secure_content_extension_wrapper.cpp

```diff
@@ -92,7 +97,7 @@ uint64_t OpenXRFbCompositionLayerSecureContentExtensionWrapper::_set_viewport_co
 
 	switch ((ExternalOutput)(int)p_property_values.get(EXTERNAL_OUTPUT_PROPERTY_NAME, EXTERNAL_OUTPUT_DISPLAY)) {
 		case EXTERNAL_OUTPUT_DISPLAY: {
-			secure_content->flags = 0;
+			// Skip.
```

Comment:
I'd mention something to the effect that we should not reach this code since it should have already been handled above.

Reply:
Good idea! Comment updated in my latest push